Resolução do P1 Simulado 01

tobiasMarion opened the pull request:
Olá prof, conforme havíamos conversado realizei a resolução do Simulado 01 da Prova 1. 
Qualquer alteração estou a disposição.


Atenciosamente, 
Tobias Cadoná Marion - (00590278)

tobiasMarion committed:
Resolução do P1 Simulado 01

schnorr commented:
Oi @tobiasMarion, obrigado pelo envio das soluções. Tu poderias:

- Nas respostas que empregam o `scanf` (ex[2-3,5-6]), capturar o retorno para ter certeza que a entrada funcionou?
- Nas respostas que empregam o `fgets` (ex4), remover o eventual `\n` que é lido para que os programas funcionem

tobiasMarion committed:
Validando entradas

Códigos refatorados para tratar possíveis erro de leituras, além de arrumar bug no ex4 envolvendo o caractere '\n'

tobiasMarion commented:
Boa noite prof, realizei as correções. Acredito que com estas alterações, os possíveis erros de leitura serão tratados. 

Novamente, qualquer alteração estou a disposição.
